.github/workflows/openhands-resolver.yml

```diff
@@ -69,7 +89,9 @@ jobs:
         contains(github.event.review.body, inputs.macro || '@openhands-agent') &&
         (github.event.review.author_association == 'OWNER' || github.event.review.author_association == 'COLLABORATOR' || github.event.review.author_association == 'MEMBER')
         )
-      )
+      ) ||
+      github.workflow_ref != ''
```

Comment:
I'm a little uncertain of including this here; for instance on Openhands we only want maintainers to run the resolver (because of LLM API costs)

Will this be behavior be preserved?